app/javascript/components/automated-workflows/summary.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Loading, Tabs, Tab, CodeSnippet,
+} from 'carbon-components-react';
+import AWSSfnGraph from '@tshepomgaga/aws-sfn-graph';
+import MiqStructuredList from '../miq-structured-list';
+import { automatedWorkflowData } from './automated-workflows-dummy-data';
+import '@tshepomgaga/aws-sfn-graph/index.css';
+
+const AutomatedWorkflowSummary = ({ recordId }) => {
+  const tabLabels = [
+    { name: 'json', text: 'JSON Data' },
```

Comment:
Ideally this shouldn't say JSON, even though the format is JSON.  Technically it's Amazon States Language or ASL.  I'm thinking it should say either `Text` or `ASL` for this tab.  @agrare Thoughts?